webui/react/src/pages/ResourcepoolDetail.tsx

```diff
@@ -197,4 +200,10 @@ const ResourcepoolDetail: React.FC = () => {
   );
 };
 
-export default ResourcepoolDetail;
+const ResourcepoolDetailPage: React.FC = () => (
+  <Suspense fallback={<Page loading />}>
+    <ResourcepoolDetail />
+  </Suspense>
+);
+
+export default ResourcepoolDetailPage;
```

Comment:
I know its working with this name, but i think the name should be `ResourcepoolDetail` or adjust the name in https://github.com/determined-ai/determined/blob/bdf5b3ec5863e450ae0d9aa03d1fdee7588a7eac/webui/react/src/routes/index.tsx#L41
Then we can find the name much easier

Reply:
Sure, I can just rename `ResourcepoolDetail` to something else.

Reply:
Changed